src/components/reservation/ReservationModalBody.tsx

```diff
@@ -141,7 +144,14 @@ export const ReservationModalBody = ({
   const reservationResult = reservationResponse?.reservationResults[0]?.result;
   const reservationDetails =
     reservationResponse?.reservationResults[0]?.reservationDetails;
-  const manifestation = selectedManifestations[0];
+  const manifestation =
+    manifestationsToReserve?.[0] || selectedManifestations[0];
+  const getEditionText = () => {
+    if (!materialIsFiction(work) || manifestationsToReserve?.length === 1) {
+      return manifestation.edition?.summary;
+    }
+    return t("firstAvailableEditionText");
+  };
```

Comment:
Is there a reason why we want to define a function at this level? It does not seem to be a pattern we use much elsewhere. I mean: Either functions are placed in `helper.ts` or the variable is defined directly as `const editionText`.